data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,484 @@
+		set "unfettered: undeserved ladybug"
+	on fail
+		"reputation: Hai (Unfettered)" >?= -1000
+		dialog `The Unfettered hail you, "You failed to complete the duel properly, and as such, are no longer authorized to land on our worlds. You will not enjoy unwarranted respect any longer!"`
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Ancient Ladybug"
+		to despawn
+			"total ships" != 1
+		personality unconstrained launching plunders staying target heroic disables
+		ship "Lightning Bug (Ionic)" "Erazer"
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Lighting"
+		ship "Lightning Bug (Pulse)" "Struck"
+		conversation
+			`You receive a message from the remaining Unfettered ships: "We surrender." You remember you were asked to land on the same day in order to claim your reward of permanent access to Unfettered space.`
+			branch "good boy or bad boy enough"
+				has "unfettred: beaten extra bugs"
+				"old reputation: Hai (Unfettered)" < -2000
+			`	"Do not shout victory too soon, however, for we were not the only ones you had to beat," they add.`
+			label "good boy or bad boy enough"
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"old reputation: Hai (Unfettered)" < -2000
+			has "flagship model: Ancient Ladybug"
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained entering plunders staying target heroic
+		ship "Sea Scorpion (Shredder)" "Requiem"
+		system "Hi Yahr"
+		dialog "You have managed to defeat the last of the Unfettered challengers, it is now time to land to claim your reward."
+		on disable
+			set "unfettred: beaten extra bugs"
+	npc
+		government "Hai (Unfettered)"
+		ship "Lightning Bug (Unfettered Shipyards)"
+		personality launching staying surveillance
+		on kill
+			fail "Hai Legacy 3"
+	on visit
+		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight. Load an earlier save and try again if you want to finish this mission."
+	to complete
+		has "unfettered: beaten the bugs"
+		has "flagship model: Ancient Ladybug"
+		or
+			"old reputation: Hai (Unfettered)" > -2000
+			has "unfettered: beaten extra bugs"
+	on complete
+		conversation
+			`The engineer seems shocked at first. "You really... won? It really was close until the end, was it not? Or did the captains already make stories to justify their incompetence?"`
+			choice
+				`	"Indeed, but it was fun."`
+					goto next
+				`	"What are you talking about? This was easy!"`
+			`	"Don't become overconfident now, captain. I am sure you know what happens to such people.`
```

Comment:
What happens?

Reply:
they get shot by a bullfrog AM, ofc

Reply:
(:

Reply:
or a ship lands on their face, you never know...